CallTracking.Web.Test/Controllers/LeadSourcesControllerTest.cs

```diff
@@ -34,56 +33,41 @@ public void Setup()
 
 
         [Test]
-        public async void Create_creates_a_lead_source()
+        public void Create_creates_a_lead_source()
         {
-            var controller = GetLeadSourcesController(_mockRepository.Object, _mockRestClient.Object);
+            var controller = new LeadSourcesController(_mockRepository.Object, _mockRestClient.Object);
+
+            controller.WithCallTo(ctrl => ctrl.Create("+1 555 555 55555"));
```

Comment:
I would prefer the use of `c` instead of `ctrl` to keep consistency with the other tests.